.github/workflows/test_imports.yml

```diff
@@ -38,8 +38,7 @@ jobs:
         shell: bash  # Necessary for pipeline to work on windows
       - name: Install in full mode
         run: |
-          python -m pip install -U pip  # Official recommended way
-          pip install -e .[full]
+          pip install .[full]
```

Comment:
what's the logic for removing -e?

Reply:
There is no reason to install editable here. Plus, the setup tools editable install is in a transition phase right now and I don't want to partake on that.

https://setuptools.pypa.io/en/latest/userguide/development_mode.html

Reply:
I see. Cool thanks.